proto/scene-format.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+
+package turbocow;
+
+message Scene {
+	enum MaterialType {
+		SOLID = 0;
+		GLASS = 1;
+		// other matherials
+	}
+	message Vector3 {
+		float x = 2;
+		float y = 3;
+		float z = 4;
+	}
+	message Transform {
+		int32 id = 5;
+		Vector3 translation = 6;
+		Vector3 rotation = 7;
+		Vector3 scale = 8;
+		int32 parent_id = 9;
+	}
+	message Cube {
+		int32 id = 34;
+		Vector3 upper_bounds = 10;
+		Vector3 lower_bounds = 11;
+		Transform transform = 11;
+	}
+	message Sphere {
```

Comment:
Please also add a plane.

Reply:
A plane is defined by two vertices?

Reply:
A plane is defined by a point and a normal. But if we using transform any of these parameters should not be specified. A point is defined by transformations and normal is the forward vector of object